### PR TITLE
Remove unused SASS and GOV mixins

### PR DESCRIPTION
**Pull request by feedmypixel**
- Remove as many mixins provided by gov as possible,
- Remove any css that is not being used by JavaScript or view files

## Of Note
The only GOV mixin I haven't managed to remove is `button`
This removes a lot of SASS work! But I am pretty confident from using a number of scripts that there shouldn't be too many styling regressions from this work. If there they can be fixed as and when.

### CODE REVIEW

**Comment by feedmypixel** [comment]
work associated with https://github.com/uktrade/data-hub-frontend/issues/226

File: assets/stylesheets/_deprecated/components/_pagination.scss [hunk 1]
```diff
@@ -1,32 +0,0 @@
-.pagination {
```
Comment (teneightfive) [inline assets/stylesheets/_deprecated/components/_pagination.scss]: There's a comment in [paginvation-nav](https://github.com/uktrade/data-hub-frontend/blob/develop/assets/stylesheets/components/_pagination-nav.scss#L5) to remove some declarations when this is gone. Might be worth checking.

File: assets/stylesheets/_deprecated/components/_searchbar.scss [hunk 2]
```diff
@@ -20,7 +19,7 @@
 
   .searchbar__input {
     box-sizing: border-box;
-    font-size: em(19);
+    font-size: 19px;
```
Comment (teneightfive) [inline assets/stylesheets/_deprecated/components/_searchbar.scss]: Do we want to use the font mixin?

Comment (feedmypixel) [inline assets/stylesheets/_deprecated/components/_searchbar.scss]: I was thinking of using [postcss-pxtorem](https://github.com/cuth/postcss-pxtorem) thoughts?

**Pull request merged**